TowerDefence_UnityProject/Assets/Scripts/TowerDefence/World/Path/Data/PathData.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Threading.Tasks;
+using UnityEngine;
+
+namespace TowerDefence.World.Path.Data
+{
+    [Serializable]
+    public class PathData
+    {
+        [JsonProperty("pathPoints")]
+        public readonly PathPoint[] pathPoints;
+
+        [JsonConstructor]
+        public PathData(PathPoint[] pathPoints)
+        {
+            this.pathPoints = pathPoints;
+        }
+    }
+
+    [Serializable]
+    public class PathPoint
+    {
+        [JsonProperty("pointId")]
+        public readonly Guid pointId;
+
+        [JsonProperty("position")]
+        public readonly Vector3 position;
+
+        [JsonProperty("pointType")]
+        public readonly PointType pointType;
```

Comment:
Use a `ItemConverterType = typeof(StringEnumConverter)` when using a enum so you can read it back easy in the json file